scripts/presets/travis/standalone/standalone-x509.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "mongod",
+    "id" : "STANDALONE_X509",
+    "authSource": "\$external",
```

Comment:
"$" is not a special character in JSON. There should be no need to escape here.

Reply:
But it was definitely necessary. It's likely to do with the way how the ``mo.sh`` tool handles sending things to Mongo Orchestration.